data/health/health-small-statistical-areas/index.html

```diff
@@ -5,45 +5,58 @@
   display_name: JP
   email: agrc@utah.gov
 date: 2011-11-21 17:25:17 -0700
-title: Small Health Statistical Areas
 categories: []
+2020SmallHealthStatisticalAreas:
+  hub:
+    name: Utah Health Small Statistical Areas 2020
+    item_id: 67edfe9ded464e3393c1932b4d39a6b2
+  updates:
+    - May 2020
 2018SmallHealthStatisticalAreas:
-  downloads:
-    - url: https://drive.google.com/drive/u/0/folders/164y4SenlrgzU-dXyPxMtKCDIbH3ZdXU9?ogsrc=32
-      label: 2018 Small Statistical Areas
+  hub:
+    name: Utah Health Small Statistical Areas 2018
+    item_id: c0a84e302c8142cb9ca34f88813f23b3
   updates:
     - October 2018
 2017SmallHealthStatisticalAreas:
-  downloads:
-    - url: https://drive.google.com/drive/u/0/folders/0B6R_P-pQXOcsQnc5bnBFajRfZzA
-      label: 2017 Small Statistical Areas
+  hub:
+    name: Utah Health Small Statistical Areas 2017
+    item_id: 6fe09518dc62415d88570dfd59fbb4c5
   updates:
     - November 2017
 2014SmallHealthStatisticalAreas:
   downloads:
-    - url: https://drive.google.com/a/utah.gov/uc?id=0ByStJjVZ7c7mYzR2TEdaTHIwa0E&export=download
-      label: '2014 Small Statistical Areas: File Geodatabase'
-    - url: https://drive.google.com/a/utah.gov/uc?id=0ByStJjVZ7c7mNlNPZU5SRVFJcVU&export=download
-      label: '2014 Small Statistical Areas: Shapefile'
+    - url: https://utah.maps.arcgis.com/home/item.html?id=30c1368e89314f34a547779772cb1d13
```

Comment:
**question**: ok these must be shelved or static and not in open data. is that why you kept the download include?

Reply:
yes 2014 is shelved